packages/components/pie-card/src/index.ts

```diff
@@ -1,9 +1,9 @@
 import {
-    html, LitElement, unsafeCSS, nothing, TemplateResult,
+    html, LitElement, unsafeCSS, nothing, TemplateResult, type PropertyValues,
```

Comment:
```suggestion
    html, LitElement, unsafeCSS, nothing, type TemplateResult, type PropertyValues,
```

Reply:
Will address in another pr if that's ok

Reply:
And could you do `CardProps` at the same time? 😄